// Copyright 2023 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

option optimize_for = LITE_RUNTIME;
option java_package = "org.chromium.components.optimization_guide.features.proto";

option java_outer_classname = "DefaultProto";

package optimization_guide.proto;

// DefaultLoggingData consists of Request, Response and Quality data.
message DefaultLoggingData {
  DefaultRequest request_data = 1;
  DefaultResponse response_data = 2;
  DefaultQuality quality_data = 3;
}

message DefaultRequest {}
message DefaultResponse {}
message DefaultQuality {}
